crates/livesplit-hotkey/src/linux/key_code.rs

```diff
@@ -658,41 +658,41 @@ impl FromStr for KeyCode {
 //     Break = 0xFF6B,
 //     ModeSwitch = 0xFF7E,
 //     NumLock = 0xFF7F,
-//     NumPadSpace = 0xFF80,
```

Comment:
What's the reason for maintaining this big list?

Reply:
I actually have no idea what this is even for either. I'll clean up the KeyCode enums (and merge them into one) in a subsequent PR.